cmd/kube-apiserver/app/server.go

```diff
@@ -197,7 +197,10 @@ func CreateServerChain(completedOptions completedServerRunOptions, stopCh <-chan
 		return nil, err
 	}
 
-	if insecureServingOptions != nil {
+	if true || completedOptions.InsecureServing.BindPort > 0 {
```

Comment:
What's the meaning of `if true || xxx` ?
Maybe add some comment here.

Reply:
hi @CaoShuFeng thanks for reviewing. i thought that it's impossible to make the server run without insecure related options. reverted the change for better human-read